documentation/source/base/design_build.dot

```diff
@@ -26,15 +26,15 @@ digraph {
     subgraph cluster_0 {
         label="Designer";
         tooltip="Designer";
-        href="../base/design_build.html"
+        href="../base/design_build.html#designers"
```

Comment:
Note that these links now go to the headers of the respective documents. I'm unsure whether we should or just link to the page as there is some overview at the top (same for the rest in this file